### PR TITLE
Provide example k8s manifests

### PR DESCRIPTION
**Pull request by kakkoyun**
This PR,

* provides example k8s manifests
* fixes image build issues
* initialize jsonnet lib

### CODE REVIEW

**Comment by kakkoyun** [comment]
@metalmatze @squat anyone? No one?

File: examples/manifests/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: observatorium-api
+    spec:
+      containers:
+      - args:
+        - --web.listen=0.0.0.0:8080
+        - --metrics.ui.endpoint=http://127.0.0.1:9091/
+        - --metrics.query.endpoint=http://127.0.0.1:9091/api/v1/query
```
Comment (metalmatze) [inline examples/manifests/deployment.yaml]: Interesting. If you allow these to be configured to they have independent listeners? 
I guess it's not really a Kubernetes manifest question though...

**Comment by squat** [comment]
It would be easier to review this PR if it was broken down into multiply smaller PRs, e.g.
* introduce image tagging
* add jsonnet tooling
* add k8s manifests

**Comment by squat** [comment]
looks good to me generally :+1:

File: examples/manifests/deployment.yaml (hunk 1 above)
Comment (kakkoyun) [reply examples/manifests/deployment.yaml]: They have a single listener but difference reverse proxies. Router dispatches requests accordingly.

https://github.com/observatorium/observatorium/blob/24b2fcd3824b9aed464a0978d942c4ec4fbb7abe/internal/server/server.go#L60

A proxy:

https://github.com/observatorium/observatorium/blob/24b2fcd3824b9aed464a0978d942c4ec4fbb7abe/internal/proxy/proxy.go#L33-L68

We'll see how it's work in real life :D

**Comment by kakkoyun** [comment]
@squat Will do next time, sorry for lengthly PR.

**Comment by kakkoyun** [comment]
@metalmatze any other concerns?

**Pull request merged**